mirgecom/initializers.py

```diff
@@ -412,8 +412,8 @@ def exact_rhs(self, discr, q, t=0.0):
         """
         Create the RHS for the lump-of-mass solution at time *t*, locations *x_vec*.
 
-        Note that this routine is only useful for testing under the condition of
-        uniform, and constant velocity field.
+        Note that *t* is used to advect the mass lumps under the assumption of
+        constant, and uniform velocity.
```

Comment:
Changed the comment to this.  Hope it helps.

Reply:
What's this responding to?